provisioner.go

```diff
@@ -183,10 +193,20 @@ func (p *defaultProvisioner) provisionPackage(ctx context.Context, paths Paths,
 	}
 
 	// Move
-	if err = m(ctx, paths); err != nil {
+	var closeFunc func() error
+	if closeFunc, err = m(ctx, paths); err != nil {
 		return fmt.Errorf("moving %s failed: %w", name, err)
 	}
 
+	// Make sure to close
+	defer func() {
```

Comment:
you need to check whether `closeFunc != nil` first

Reply:
done. https://github.com/asticode/go-astilectron/pull/358/commits/4bac7b9d74646f5eec8c4f79dee7a2754e9d241c#diff-752145c98ffd12d3a2b292517f8cf139ca3bcaa6512dc88dd7d8628372fd5df4R203-R205